### ISSUE TITLE
Scroll not working still.. (and bug for back helper repeating itself..) 

### ISSUE THREAD

**Issue by willwade** [AceCentre/pasco#178]
Ok - I swear this did work a bit - but only on second level items - and then it was quite jerky... But then when I've gone to do this video - I cant get it working at all.. 

Hmmm
https://www.dropbox.com/s/m5fl5e4f7ygasom/RPReplay_Final1565700016.MP4?dl=0

**Comment by hosseinamin** [comment]
It maybe that when you connect mouse the webview have tricked us as it is not ios platform. To test fake-scroll directly I've made a button (when debug mode (buttons) are on it is visible) Fake Scroll Toggle.

Also I did another quick commit. For adding dot (.) to list of keycommands for us to capture dot.
Please pull my commit from repo and hit dot (.) to bring up debug keys, Then you will be able to test both modes. fake-scroll or wheel event.

```
git remote add hossein https://github.com/hosseinamin/pasco
git pull hossein master
```

**Comment by willwade** [comment]
Ok - Done that - for some reason that p or . aren't being picked up. It's not changing anything (should it?) NB: 1.1.3/105 on testflight is your repo build. 

Setup: iOS 13, running hossein master and using a bluetooth keyboard.

Weirdly I'm also getting a new bug where the back helper keeps getting added:

https://www.dropbox.com/s/nx3q49ei15xkhv2/RPReplay_Final1566292680.MP4?dl=0

**Comment by hosseinamin** [comment]
I couldn't yet figure out why multiple back option pops up. If you could come up with steps for generating this bug. That would be great.

**Comment by hosseinamin** [comment]
multi-back bug has fixed in PR #180

**Comment by willwade** [comment]
Fab - I can see the multi-back bug fixed in #180
